programs/degen-pools/src/constants.rs

```diff
@@ -1,2 +1,2 @@
 use anchor_lang::prelude::*;
-pub const AUTHORITY_PUBKEY: Pubkey = pubkey!("rv9MdKVp2r13ZrFAwaES1WAQELtsSG4KEMdxur8ghXd");
+pub const AUTHORITY_PUBKEY: Pubkey = pubkey!("2JWqYTXG5yHSU78hjKb39YFx82whbK74v6sMqMG3TVBQ");
```

Comment:
We should never commit this as this wallet is our Degen Markets team wallet

Reply:
Apologies for the oversight, In the last PR (pause-pool).
after Last PR merged, 
I was testing locally and forgot to revert the main addresses before submitting.
I did add degen_pool.json from S3, but missed updating the addresses back to the correct ones.

I'll make sure to be more careful with this in future PRs.

Reply:
Thanks!